src/Sarif.Viewer.VisualStudio/Services/LoadSarifLogService.cs

```diff
@@ -49,6 +53,8 @@ public void LoadSarifLogs(IEnumerable<string> paths)
         /// <inheritdoc/>
         public void LoadSarifLogs(IEnumerable<string> paths, bool promptOnSchemaUpgrade)
         {
+            Telemetry.Events.LoadSarifLogApiInvoked();
```

Comment:
>LoadSarifLogApiInvoked [](start = 29, length = 22)

Shouldn't this have arguments, at least the `path`?

Reply:
Come to think of it `path` for the first two overloads, `paths` (perhaps string-joined?) for this one.

---
In reply to: [505741594](https://github.com/microsoft/sarif-visualstudio-extension/pull/226#discussion_r505741594) [](ancestors = 505741594)

Reply:
No, we absolutely cannot send the path as it will almost always have a userid in it.

---
In reply to: [505742115](https://github.com/microsoft/sarif-visualstudio-extension/pull/226#discussion_r505742115) [](ancestors = 505742115,505741594)